sphinx_needs/filter_common.py

```diff
@@ -299,6 +299,10 @@ def filter_needs(
     return found_needs
 
 
+def need_search(*args, **kwargs):
+    return bool(re.search(*args, **kwargs))
```

Comment:
```suggestion
    return re.search(*args, **kwargs) is not None
```

this is maybe a bit more specific?